pyomo/util/subsystems.py

```diff
@@ -143,22 +189,31 @@ def __init__(self,
                       of values of length n_scenario
         output_values: ComponentMap mapping each output variable to a list
                        of values of length n_scenario
+        to_fix: to_fix argument for base class
+        to_deactivate: to_deactivate argument for base class
+        to_reset: to_reset argument for base class. This list is extended
+                  with input variables.
```

Comment:
Can you reformat this to include the argument type?

Reply:
Updated